.circleci/config.yml

```diff
@@ -38,4 +38,32 @@ jobs:
           destination: reports
       - store_artifacts:
           path: build/reports/jacoco/jacocoFullReport/
+  push_translations:
+    docker:
+      - image: circleci/python:3.6
+    working_directory: ~/ultrasonic
+    steps:
+      - checkout
+      - run:
+          name: install transifex client
+          command: |
+            python -m venv ~/venv
+            . ~/venv/bin/activate
+            pip install transifex-client
+      - run:
+          name: configure transifex client
+          command: echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = api\npassword = '"${TRANSIFEX_PASSWORD}"$'\n' > ~/.transifexrc
+      - run:
+          name: push changes in translation files
+          command: |
+            . ~/venv/bin/activate
+            tx push -st
+workflows:
+  version: 2
+  build_and_push_translations:
+    jobs:
+      - build
+      - push_translations:
+          requires:
+            - build
```

Comment:
Not sure that `build` here is right one job 🤔 

I think better to use a separate job that builds release type, but for now it is ok.

Reply:
I think that better workflow is sequential like this:
1. Do tests.
2. Build.
3. Push translations (because the project build well).
4. If is a tag `/^[0-9]*\.[0-9]*\.[0-9]*$/` sing build and publish it in release.